dqc_us_rules/dqc_us_0013.py

```diff
@@ -0,0 +1,135 @@
+# (c) Copyright 2015 - 2016, XBRL US Inc. All rights reserved.
+# See license.md for license information.
+# See PatentNotice.md for patent infringement notice.
+import os
+from .util import facts, messages, neg_num
+import decimal
+
+_CODE_NAME = 'DQC.US.0013'
+_RULE_VERSION = '1.1'
+_DEFAULT_CONCEPTS_FILE = os.path.join(
+    os.path.dirname(__file__),
+    'resources',
+    'DQC_US_0013',
+    'dqc_13_concepts.csv'
+)
+# The same exclusion rules used in Rule 15 also apply to this rule
+_DEFAULT_EXCLUSIONS_FILE = os.path.join(
+    os.path.dirname(__file__),
+    'resources',
+    'DQC_US_0015',
+    'dqc_15_exclusion_rules.csv'
+)
+
+_PRECONDITION_ELEMENT_1 = 'IncomeLossFromContinuingOperationsBeforeIncomeTaxesExtraordinaryItemsNoncontrollingInterest'
```

Comment:
Nevermind... I thought it would help things out below, but with how the rule is written, this will only make things more difficult.